### PR TITLE
Not Eligible page function - Ticket FFTF -684

### PR DESCRIPTION
**Pull request by ellietms**
added function for not eligible options in pages / added elimination page for project-start page

### CODE REVIEW

**Review by diya2214 — CHANGES_REQUESTED** [review]
fix the url part in ineligible object of all questions

File: app/config/question-bank.js [hunk 1]
```diff
@@ -253,8 +262,11 @@ module.exports = {
           url: 'project-start',
           backLink: 'planning-permission',
           nextUrl: 'tenancy',
-          eliminationAnswerKeys: '',
-          ineligibleContent: { heading: '', message: '', link: '' },
+          ineligibleContent: {
+            messageContent: 'You cannot apply for a grant if you have already started work on the project.',
+            insertText: { text: 'Starting the project or committing to any costs (such as placing orders) before you receive a funding agreement invalidates your application.' },
+            messageLink: 'https://www.gov.uk/government/collections/rural-payments-and-grants'
```
Comment (diya2214) [inline app/config/question-bank.js:268]: convert it into object

File: app/config/question-bank.js [hunk 2]
```diff
@@ -253,8 +266,13 @@ module.exports = {
           url: 'project-start',
           backLink: 'planning-permission',
           nextUrl: 'tenancy',
-          eliminationAnswerKeys: '',
-          ineligibleContent: { heading: '', message: '', link: '' },
+          ineligibleContent: {
+            messageContent: 'You cannot apply for a grant if you have already started work on the project.',
+            insertText: { text: 'Starting the project or committing to any costs (such as placing orders) before you receive a funding agreement invalidates your application.' },
+            messageLink: {
```
Comment (diya2214) [inline app/config/question-bank.js:272]: can you change the object like this 
{
      url: '',
      title: ''
    }

File: app/routes/index.js [hunk 3]
```diff
@@ -83,6 +83,19 @@ const getModel = (data, question) => {
   }
   return model
 }
+
+function createModelNotEligible (backUrl, ineligibleContent) {
+  return {
+    backLink: backUrl,
+    messageContent: ineligibleContent.messageContent,
+    insertText: ineligibleContent.insertText,
+    messageLink: {
```
Comment (diya2214) [inline app/routes/index.js:92]: this should be 
      messageLink: ineligibleContent.messageLink

**Review by diya2214 — CHANGES_REQUESTED** [review]

File: app/routes/index.js (hunk 3 above)
Comment (ellietms) [reply app/routes/index.js:92]: it is not working for title if I just write this : messageLink: ineligibleContent.messageLink

**Review by diya2214 — APPROVED** [review]
LGTM

**Pull request merged**